python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/monitor-server-radius.j2

```diff
@@ -0,0 +1,25 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - monitor server radius #}
+{% if monitor_server_radius is arista.avd.defined %}
+!
+monitor server radius
+{%     if monitor_server_radius.service_dot1x is arista.avd.defined(true) %}
+   service dot1x
+{%     endif %}
+{%     if monitor_server_radius.probe.interval is arista.avd.defined %}
+   probe interval {{ monitor_server_radius.probe.interval }} seconds
+{%     endif %}
+{%     if monitor_server_radius.probe.threshold_failure is arista.avd.defined %}
+   probe threshold failure {{ monitor_server_radius.probe.threshold_failure }}
+{%     endif %}
+{%     if monitor_server_radius.probe.method is arista.avd.defined("status-server") %}
+   probe method status-server
+{%     elif monitor_server_radius.probe.method is arista.avd.defined("access-request") %}
+{%         set access_request = monitor_server_radius.probe.access_request %}
+   probe method access-request username {{ access_request.username }} password {{ access_request.password_type }} {{ access_request.password }}
```

Comment:
what if password_type is not defined

Reply:
Can you please check the coverage and try to improve

Reply:
it raises an error
```
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: jinja2.exceptions.UndefinedError: 'ansible.parsing.yaml.objects.AnsibleMapping object' has no attribute 'password_type'
fatal: [monitor-server-radius-1 -> localhost]: FAILED! => {"changed": false, "msg": "Error during plugin execution: 'ansible.parsing.yaml.objects.AnsibleMapping object' has no attribute 'password_type'"}
```

EoS allows to set `username` and `password` without specifying type of the password. In this case it treats it as type "0" (cleartext). Therefore if `username` and `password` are set but `type` is not set - we should default to type `0` (cleartext).